analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -280,7 +281,7 @@ private fun Collection<Edge>.toPackageReferenceForest(
 private val PSEUDO_VERSION_REGEX = "^v0.0.0-(?:[\\d]{14}-(?<sha1>[0-9a-f]+)$)".toRegex()
 
 private fun getRevision(version: String): String {
-    if (version.endsWith("+incompatible")) return getRevision(version.removeSuffix("+incompatible"))
+    version.withoutSuffix("+incompatible")?.let { return getRevision(it) }
```

Comment:
I found the original version easier to read and understand; the new variant is certainly more elegant. So I won't object here.

Reply:
It's probably just again one of those things one needs to get used to 😀 Thanks for approving anyway.